### DISCUSSION TITLE
Conscious Coliving in Action: Practical Steps to Getting Started - an outline of a course

### DISCUSSION THREAD

**Discussion by laurenwigmore** [life-itself/community#891] [Ideas]
This discussion thread is following on from our recent poll where we asked what topics would you like in our follow up course to [Conscious Coliving 101](https://lifeitself.org/conscious-coliving-course). As a result of this discussion here is our draft skeleton for a *Conscious Coliving in Action: Practical Steps to Getting Started with Conscious Coliving* email course. 

We'd love further ideas and input to help us shape it!

### 1 So you’re interested …
- Brief recap of what CC is
- What the spectrum of it can look like (i.e it is NOT (necessarily) living in a commune in rural Missouri!)

#### Spectrum of living options sketch
![Spectrum of Living Options](https://github.com/life-itself/community/assets/132304704/74a5c7ef-9df6-43b2-8a24-4fa083490363)

### 2 The 3 options available
- Start with baby-steps. This doesn’t mean quitting your job and going to rural Missouri. It doesn’t mean having to start from scratch (e.g. you can join something existing for a trial week, month, visit)
    - You can move between these options and do several at once
    - Example of woman who came to a Life Itself residency with her potential co-founder. Ultimately realized they didn’t want to start a Coliving Community (at least together!). She said this was so useful and much better than realizing when they were 1y into their project)
- Finding a flatmate or starting your own coliving experiment
    - Get more conscious (in your coliving): already have flatmates or in “coliving” but perhaps it's not really so conscious. Here’s how you can move towards conscious coliving … [Could be 2 episodes or a follow on episode later]
 - How do i get buy-in from current colivers?

This sketch sets out the main options for "next steps to conscious coliving" for people who have an interest but aren't yet doing it.
![Practical Action overview](https://github.com/life-itself/community/assets/132304704/40d1ee44-3a7f-4fa8-8ae2-5b93876fee48)
### 3 Time to trial it as an option

- Things to bear in mind - warning: “Old habits die hard”
- Here’s how to find some (here’s some we recommend)

### 4 Next steps to take
- Making a commitment and what might come up in the process i.e. internal resistance, magpie vision
   - But it’s easier than you think
   - And it’s fun
   - And you’ll learn a lot
   - Commit

**Comment by rufuspollock** [comment]
Here are some diagrams we generated that may be useful. /cc @laurenwigmore (maybe you want to incorporate into the overview)

### Next steps 

This sketch sets out the main options for "next steps to conscious coliving" for people who have an interest but aren't yet doing it.

![image](https://github.com/life-itself/community/assets/180658/72e36384-439d-4aaf-a420-a300a17e7421)

### Spectrum of living options (WIP)

![image](https://github.com/life-itself/community/assets/180658/f375b5bd-eb52-4432-9b80-994020342a4a)
